File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner s = new Scanner(System.in);
		
		int N = s.nextInt();
		
		long number = 0;
		long count = 0;
		while(count != N) {
			number++;
			String str = String.valueOf(number);
			if(str.contains("666")) {
				count++;
			}
		}
		
		System.out.println(number);
	}
}

File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.kt
fun main() {
    val N = readln().toInt()
    
    var number = 0
    var count = 0
    while(count != N) {
        number++
        val str = number.toString()
        if(str.contains("666")) {
            count++;
        }
    }
    
    println(number)
}